### ISSUE TITLE
BaseVolume and QuoteVolume are reversed on poloniex

### ISSUE THREAD

**Issue by robkaandorp** [ccxt-dev/ccxt#292]
On bittrex the BaseVolume seems to be the BTC volume, whereas on poloniex the QuoteVolume seems to be the BTC volume. I noticed this in the parseTicker code for poloniex:

```
'baseVolume': parseFloat (ticker['quoteVolume']),
'quoteVolume': parseFloat (ticker['baseVolume']),
```

Should I create a pull-request for this?

Regards,
Rob

**Comment by kroitor** [comment]
@robkaandorp hi! Thanks for opening this issue! This must be related to #284 . I'll fix for that. By the way, your PRs are welcome! ) Thx again!

**Comment by kroitor** [comment]
Hm... actually:

```
>>> pprint (polo.fetch_ticker ('ETH/BTC'))
{'ask': 0.06559499,
 'average': None,
 'baseVolume': 62697.24810407, # ← in ETH
 'bid': 0.06556325,
 'change': -0.04934463,
 'close': None,
 'datetime': '2017-10-09T14:16:44.000Z',
 'first': None,
 'high': 0.06908309,
 'info': { ... },
 'last': 0.06559522,
 'low': 0.065281,
 'open': None,
 'percentage': None,
 'quoteVolume': 4194.9708818, # ← in BTC
 'timestamp': 1507558604993,
 'vwap': None}
```

If you look closer at baseVolume and quoteVolume, as well as the ETH/BTC pair in general, you'll see that:

1. ETH is the base, BTC is quote in this pair (base is always before the slash)
2. quoteVolume in BTC is much lower that baseVolume in ETH
3. quoteVolume in BTC / baseVolume in ETH = 4194.9708818 / 62697.24810407 = 0.06690837331	(approx market price).
So 1 ETH of base currency costs ~ 0.06 of BTC quote currency.
In other words, the quoted BTC price of base-ETH is 0.06.

So, ETH is the base and BTC is quote and the ticker looks correct? What do you think about it, @robkaandorp ?

**Comment by robkaandorp** [comment]
So maybe the bittrex code is wrong then? @kroitor

**Comment by kroitor** [comment]
@robkaandorp it may be, but I don't think so. 

The hint: Bittrex incorrectly named `base` as BTC in the `ETH/BTC` pair, whereas the actual base is ETH (base is always before the slash). 50% of their users get confused by that. 

They named the pair `BTC-ETH`, but they are actually showing the price for `ETH-BTC`, confusing everyone. So, it's not the ccxt, but the Bittrex itself is inconsistent with the rest of the world. Does this make more sense? )

**Comment by kroitor** [comment]
@robkaandorp nope, I'm sorry. Looks like bittrex ticker **volumes** are still incorrect in ccxt, I'll fix for that, thx for pointing that out.

**Comment by kroitor** [comment]
Ok, this is fixed for Bittrex now! The new fixed version 1.9.72 will be available in 5 minutes. Thx once again! Let us know if you find any other thing suspicious. Cheers!

**Comment by robkaandorp** [comment]
Thank you!

**Comment by kroitor** [comment]
@robkaandorp no, thx for your attention! Really appreciate it!
